conans/client/packager.py

```diff
@@ -15,7 +15,7 @@
 def export_pkg(conanfile, package_id, src_package_folder, package_folder, hook_manager,
                conanfile_path, ref):
     mkdir(package_folder)
-    conanfile.package_folder = src_package_folder
```

Comment:
We will break someone.

Reply:
Take it for granted, but IMHO it is a bug: right now the `package_folder` for the `pre-package-hook` doesn't have the same meaning for the local commands (source folder) and for the `create` one (target folder).

If you consider it is not a bug but a feature, then feel free to close this PR and I think we should document this behavior, at least I find it quite unexpected.

Reply:
I agree it sounds weird. Could we add a new parameter to the `pre_package` with the `src_package_folder`?

Reply:
...but in the `conan create ...` flow there is no such thing as a _single_ `src_package_folder`

Reply:
It can be None in that case. Is just a matter of documenting the parameter. My concern is, if we "fix" this, users won't be able anymore to access the origin folder and that is breaking. 

Anyway, let's accept this. I don't really think someone is using a hook for the export-pkg right now.